src/tools/tidy/src/triagebot.rs

```diff
@@ -0,0 +1,24 @@
+use crate::walk::{filter_dirs, walk};
+use serde::Deserialize;
+use std::path::Path;
+use toml::Value;
+
+#[derive(Deserialize)]
+struct Config {}
```

Comment:
This seems to be unused. Can you remove it and remove the serde dependency and serde feature of the toml dependency?